buildkite/bazel-bench/bazel_bench.py

```diff
@@ -156,16 +156,39 @@ def _ci_step_for_platform_and_commits(bazel_commits, platform, project, extra_op
             "--platform=%s" % platform,
             "--collect_memory",
             "--data_directory=%s" % DATA_DIRECTORY,
+            "--csv_file_name=%s" % BAZEL_BENCH_RESULT_FILENAME,
+            "--collect_json_profile",
+            "--aggregate_json_profiles",
             extra_options,
             "--",
             project["bazel_command"],
         ]
     )
+    # TODO(leba): Upload to BigQuery too.
 
+    # Upload everything under DATA_DIRECTORY to Storage.
+    # This includes the raw data, aggr JSON profile and the JSON profiles
+    # themselves.
+    storage_subdir = "{}/{}/{}/".format(
+        project["storage_subdir"], date.strftime("%Y/%m/%d"), platform
+    )
```

Comment:
Add a TODO here to replace the gsutil command with the library call?

Reply:
Done.